docs/python/how-to-guides/use-uris.md

```diff
@@ -86,12 +130,51 @@ resolved_table = resolve("dh+plain://table-producer/scope/my_table")
 
 By resolving the URI, we acquire `my_table` from the `table-producer` container using the syntax given above.
 
+## Table scopes and paths
+
+In Deephaven, tables exist within specific scopes (namespaces) that determine how they're accessed:
+
+- `scope` - The default scope for tables created in the main session.
+- `app/<app_name>` - For tables created within a specific [Application Mode](./application-mode.md) namespace.
+- `session/<session_id>` - For tables associated with a specific user session.
```

Comment:
- What is the `main session`?
- Have we tested this (`app/<app_name>`)? If not, we should test it to make sure it's correct.
- I don't know what a `specific user session` is in the context of DHC.